devo/api/client.py

```diff
@@ -594,7 +615,7 @@ def _get_headers(self, data):
                 'Authorization': "jwt %s" % self.auth.get("jwt")
             }
 
-        raise DevoClientException(_format_error(ERROR_MSGS['no_auth']))
+        raise DevoClientException((ERROR_MSGS['no_auth']),400)
```

Comment:
If we don't have address, we cannot create an HTTP request and no HTTP status code can be returned. It will create confussion in developer/use as no HTTP service can have returned any status code. Do not create HTTP status codes. Just returns the one returned by Serrea.